dm/dumpling/dumpling_test.go

```diff
@@ -129,6 +129,32 @@ func (t *testDumplingSuite) TestDefaultConfig(c *C) {
 	c.Assert(dumpling.dumpConfig.Rows, Not(Equals), export.UnspecifiedSize)
 }
 
+func (d *testDumplingSuite) TestCallStatus(c *C) {
```

Comment:
```suggestion
func (t *testDumplingSuite) TestCallStatus(c *C) {
```

CI shows a lint error